contracts/interfaces/IPoolMEV.sol

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+
+pragma solidity 0.7.5;
+
+/**
+ * @dev Interface of the PoolMEV contract.
+ */
+interface IPoolMEV {
```

Comment:
Let's rename the file, interface, and the contract to `FeesEscrow`

Reply:
- [x] Done